src/search/FileFilters.js

```diff
@@ -490,12 +491,13 @@ define(function (require, exports, module) {
         _picker.on("listRendered", _handleListRendered);
         
         _picker.on("select", function (event, item, itemIndex) {
+            // trigger a queryChange when dropdown is selected
+            currentFindBar.trigger("queryChange");
```

Comment:
'querychange' checks for other things like validity of searchText. May be a better idea to dispatch a different event and handle it by calling _findBar.showError(null) in FindInFilesUI;